code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -279,3 +279,27 @@
 		slot_equipment_priority.Insert(index, slot_head)
 		return
 	return ..()
+
+/obj/item/weapon/reagent_containers/glass/beaker/waterbottle
+	name = "bottle of water"
+	desc = "A bottle of water filled at an old Earth bottling facility,"
+	icon = 'icons/obj/drinks.dmi'
+	icon_state = "smallbottle"
+	item_state = "bottle"
+	list_reagents = list("water" = 48, "fluorine" = 2)//see desc, don't think about it too hard

```

Comment:
It's two units of it? I don't think it's really that harmful ingame. And in real life, there is significant evidence that fluoride causes potential harmful effects (not a conspiracy theory).
I used fluorine because we don't actually have fluoride itself and it's close enough